cmd/prometheus/main.go

```diff
@@ -94,233 +92,118 @@ func main() {
-		SetValue(&newFlagRetentionDuration)
-
-	a.Flag("storage.tsdb.retention.size", "[EXPERIMENTAL] Maximum number of bytes that can be stored for blocks. Units supported: KB, MB, GB, TB, PB. This flag is experimental and can be changed in future releases.").
-		BytesVar(&cfg.tsdb.MaxBytes)
-
-	a.Flag("storage.tsdb.no-lockfile", "Do not create lockfile in data directory.").
-		Default("false").BoolVar(&cfg.tsdb.NoLockfile)
-
-	a.Flag("storage.tsdb.allow-overlapping-blocks", "[EXPERIMENTAL] Allow overlapping blocks, which in turn enables vertical compaction and vertical query merge.").
-		Default("false").BoolVar(&cfg.tsdb.AllowOverlappingBlocks)
-
-	a.Flag("storage.tsdb.wal-compression", "Compress the tsdb WAL.").
-		Default("false").BoolVar(&cfg.tsdb.WALCompression)
-
-	a.Flag("storage.remote.flush-deadline", "How long to wait flushing sample on shutdown or config reload.").
-		Default("1m").PlaceHolder("<duration>").SetValue(&cfg.RemoteFlushDeadline)
-
-	a.Flag("storage.remote.read-sample-limit", "Maximum overall number of samples to return via the remote read interface, in a single query. 0 means no limit. This limit is ignored for streamed response types.").
-		Default("5e7").IntVar(&cfg.web.RemoteReadSampleLimit)
-
-	a.Flag("storage.remote.read-concurrent-limit", "Maximum number of concurrent remote read calls. 0 means no limit.").
-		Default("10").IntVar(&cfg.web.RemoteReadConcurrencyLimit)
-
-	a.Flag("storage.remote.read-max-bytes-in-frame", "Maximum number of bytes in a single frame for streaming remote read response types before marshalling. Note that client might have limit on frame size as well. 1MB as recommended by protobuf by default.").
-		Default("1048576").IntVar(&cfg.web.RemoteReadBytesInFrame)
-
-	a.Flag("rules.alert.for-outage-tolerance", "Max time to tolerate prometheus outage for restoring \"for\" state of alert.").
-		Default("1h").SetValue(&cfg.outageTolerance)
-
-	a.Flag("rules.alert.for-grace-period", "Minimum duration between alert and restored \"for\" state. This is maintained only for alerts with configured \"for\" time greater than grace period.").
-		Default("10m").SetValue(&cfg.forGracePeriod)
-
-	a.Flag("rules.alert.resend-delay", "Minimum amount of time to wait before resending an alert to Alertmanager.").
-		Default("1m").SetValue(&cfg.resendDelay)
-
-	a.Flag("alertmanager.notification-queue-capacity", "The capacity of the queue for pending Alertmanager notifications.").
-		Default("10000").IntVar(&cfg.notifier.QueueCapacity)
-
-	a.Flag("alertmanager.timeout", "Timeout for sending alerts to Alertmanager.").
-		Default("10s").SetValue(&cfg.notifierTimeout)
+	// TODO(bwplotka): This flag is not used... by accident? Should we remove?
```

Comment:
Any ideas what happened here? We have this but it was never used... :scream:

Reply:
or we should use it :)

Reply:
hah yea